### PR TITLE
Add Card Tiles in Accordions for Home Page

### PR DESCRIPTION
**Pull request by avdgupta**

### CODE REVIEW

File: src/containers/Home/CardBox.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import Card from "@mui/material/Card";
+import CardMedia from "@mui/material/CardMedia";
+import CardContent from "@mui/material/CardContent";
+import PhoneInTalkIcon from "@mui/icons-material/PhoneInTalk";
+import PersonIcon from "@mui/icons-material/Person";
+import EmailIcon from "@mui/icons-material/Email";
+import List from "@mui/material/List";
+import { ListItem, ListItemIcon } from "@mui/material";
+
+export interface CardBoxProps {
```
Comment (PramodaM4444) [inline src/containers/Home/CardBox.tsx]: Can you pls move this to types folder

File: src/containers/Home/CardBoxList.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        email: "xyz@test.com",
+        phone: "999999999",
+    },
+    {
+        image: "/test.jpg",
+        name: "Test5",
+        email: "xyz@test.com",
+        phone: "999999999",
+    },
+];
```
Comment (PramodaM4444) [inline src/containers/Home/CardBoxList.tsx]: Pls, note that this needs to be replaced with API response, once we are done with the API integration.

File: src/containers/Home/CardBoxList.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            bgcolor: "#ffff",
+            display: "flex",
+            flexDirection: "row",
+            padding: 0,
+            width: "100%",
+            overflowX: "scroll",
+        }}
+        component="nav"
+    >
+        {cardListArray.map((item) => (
```
Comment (PramodaM4444) [inline src/containers/Home/CardBoxList.tsx]: You can destructure the "item" like below:

cardListArray.map(( { image, name, email, phone }) => ())

File: src/containers/Home/dashboardHome.tsx [hunk 4]
```diff
@@ -59,7 +59,7 @@ export const DashboardHome = () => {
                             {UIConstants.teamMemberTitle}
                         </Typography>
                     </AccordionSummary>
-                    <AccordionDetails>test 1</AccordionDetails>
+                    <AccordionDetails>{CardBoxList}</AccordionDetails>
```
Comment (PramodaM4444) [inline src/containers/Home/dashboardHome.tsx]: CardBoxList is a component. Pls import it like <CardBoxList />

**Review by PramodaM4444 — CHANGES_REQUESTED** [review]
Please have a look on the responsive design. Looks like the images are overflowing in smaller screens.

File: src/containers/Home/CardBoxList.tsx (hunk 2 above)
Comment (avdgupta) [reply src/containers/Home/CardBoxList.tsx]: Okay, noted.

File: src/containers/Home/CardBoxList.tsx (hunk 3 above)
Comment (avdgupta) [reply src/containers/Home/CardBoxList.tsx]: Done

File: src/containers/Home/dashboardHome.tsx (hunk 4 above)
Comment (avdgupta) [reply src/containers/Home/dashboardHome.tsx]: Done

File: src/containers/Home/CardBox.tsx (hunk 1 above)
Comment (avdgupta) [reply src/containers/Home/CardBox.tsx]: Done

**Review by PramodaM4444 — CHANGES_REQUESTED** [review]
Other changes look good. Pls work on the responsive design.

image

File: src/containers/Home/CardBox.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+export const CardBox: React.FC<CardBoxProps> = ({
+    cardImage,
+    cardEmail,
+    cardName,
+    cardPhone,
+}) => {
+    return (
+        <Card sx={{ width: 250, height: 350, margin: 2 }}>
+            {}
```
Comment (PramodaM4444) [inline src/containers/Home/CardBox.tsx:20]: I don't think we need { }.

**Review by PramodaM4444 — APPROVED** [review]

**Pull request merged**